django/core/management/commands/makemessages.py

```diff
@@ -399,6 +399,11 @@ def write_po_file(self, potfile, locale):
             msgs, errors, status = popen_wrapper(args)
             if errors:
                 if status != STATUS_OK:
+                    self.stdout.write(errors) # bug exception bellow not shows errors
```

Comment:
You could combine "errors" and content of `pofile` in a file.

Here is a untested approach:

```py
logfile_format = """\
Invalid input:

%s

-----

Output of %r:

%s
"""

pofile_log = '%s.log' % pofile

with open(pofile_log, "w") as fobj, open(pofile) as fpofile:
    fobj.write(logfile_format % (errors, pofile, fpofile.read()))
```